ui/pages/confirmations/components/confirm/info/typed-sign-v1/typed-sign-v1.tsx

```diff
@@ -8,21 +7,21 @@ import {
 } from '../../../../../../components/app/confirm/info/row';
 import { RowAlertKey } from '../../../../../../components/app/confirm/info/row/constants';
 import { useI18nContext } from '../../../../../../hooks/useI18nContext';
-import { currentConfirmationSelector } from '../../../../../../selectors';
 import {
   SignatureRequestType,
   TypedSignDataV1Type,
 } from '../../../../types/confirm';
+import { useConfirmContext } from '../../../../context/confirm';
 import { ConfirmInfoRowTypedSignDataV1 } from '../../row/typed-sign-data-v1/typedSignDataV1';
 import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
 
 const TypedSignV1Info: React.FC = () => {
   const t = useI18nContext();
-  const currentConfirmation = useSelector(
-    currentConfirmationSelector,
-  ) as SignatureRequestType;
+  const { currentConfirmation } = useConfirmContext() as {
+    currentConfirmation: SignatureRequestType;
+  };
 
-  if (!currentConfirmation?.msgParams) {
+  if (!(currentConfirmation as SignatureRequestType)?.msgParams) {
```

Comment:
Nit: This type may not be needed given it's already used above

Reply:
good point this is indeed not needed, I will fix this in following PR